docs/source/sim_overview.rst

```diff
@@ -1,43 +1,41 @@
 Overview
 ========
 
+.. role:: strike
+
+Why run a simulation?
+---------------------
+
+Doing simulations can be a great way to assess how well ASReview performs for your particular purposes. The user can run simulations on previously labelled datasets to see - for example - how much time the user could have saved using ASReview, or how previous inclusion criteria could be improved in the future. The ASReview simulation mode works by sequentially going through the dataset like an ASReview user would, and uses the labels to learn from these inclusions and exclusions. In this way, the entire screening process is replicated.
```

Comment:
I don't think "- for example -" is necessary.

I don't understand what you mean with "or how previous inclusion criteria could be improved in the future"? How? Maybe we need to be more specific?

I would remove the "sequentially" (since it's not?). How about:
"The ASReview simulation mode works through the dataset exactly like an ASReview user would, using the inclusions and exclusions provided in the dataset for the active learning cycle."

Reply:
The goal of the sentence is to give an example of that seeing how much time can be saved is not the only reason for using simulation mode. What is meant that we can simulate how difficult or easy it is too find certain papers and use that information to verify the screening process (maybe some inclusions shouldn't be inclusions, maybe some exclusions shouldn't be exclusions). What do you think? 

About the sentence:

"The ASReview simulation mode works through the dataset exactly like an ASReview user would, using the inclusions and exclusions in the dataset to train the model in the active learning cycle."

Reply:
Well, I think this is the place to put the other reasons to run a simulation then. Perhaps if you make it longer, you can also split up "The ASReview simulation works .." to its own section, since it's not really the "why"? Your proposed new sentence seems fine to me.